packages/account/src/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -0,0 +1,142 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { DesktopWrapper, MobileWrapper, Loading, DataTable } from '@deriv/components';
+import { parseUA } from '@deriv/shared/utils/browser';
+import { convertDateFormat } from '@deriv/shared/utils/date';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { WS } from 'Services/ws-methods';
+import LoadErrorMessage from 'Components/load-error-message';
+
+const Td = ({ title, text, className }) => (
+    <td>
+        <h3 className='cell-title'>{title}</h3>
+        <span className={classNames('cell-value', className)}>{text}</span>
+    </td>
+);
+
+const LoginHistoryListView = ({ fields, login_history }) => (
+    <table className='login-history-table'>
+        <tbody>
+            <tr>
+                <Td title={fields.date} text={login_history.date} />
+            </tr>
+            <tr>
+                <Td title={fields.browser} text={login_history.browser} />
+                <Td className='action' title={fields.action} text={login_history.action} />
+            </tr>
+            <tr>
+                <Td title={fields.ip} text={login_history.ip} />
+                <Td title={fields.status} text={login_history.status} />
+            </tr>
+        </tbody>
+    </table>
+);
+
+class LoginHistory extends React.Component {
+    fields = {
+        date: localize('Date and time'),
```

Comment:
it would be safer to use `<Localize` element so we are sure it gets re-rendered on init of translations. constants that call localize() might not get translated as init of translations will be called after. functions and `<Localize` tag don't have this issue.